### DISCUSSION TITLE
expose config schema in `@changesets/cli`

### DISCUSSION THREAD

**Discussion by unional** [changesets/changesets#1429] [Ideas]
Currently, we can specify the config schema through `unpkg.com`:

```
  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json"
```

The drawback is that it needs to be manually update.

A better approach is provide it in `@changesets/cli` and reference it with relative path:

```
  "$schema": "../node_modules/@changesets/cli/schema.json"
```

**Comment by Sec-ant** [comment]
FYI if you're using pnpm, there's a workaround if you add this to `.npmrc`:

```ini
public-hoist-pattern[]=@changesets/config
```

Then you can use the relative path for the schema in `config.json`

```json
{
  "$schema": "../node_modules/@changesets/config/schema.json"
}
```

**Reply by unional** [reply]
That's a pretty awesome workaround. 🍻
